docs/languages/julia.md

```diff
@@ -37,7 +37,7 @@ BridgeStan is registered on JuliaRegistries each release.
 ```
 
 
-The first time you compile a model, the BridgeStan source code will be downloaded as an [Artifact](https://pkgdocs.julialang.org/v1/artifacts/). If you prefer to use a source distribution of BridgeStan, consult the following section.
+The first time you compile a model, the BridgeStan source code for your current version will be downloaded as an [Artifact](https://pkgdocs.julialang.org/v1/artifacts/). If you prefer to use a source distribution of BridgeStan, consult the following section.
```

Comment:
Can we add a sentence like (copied from the `r.md` file) the following to the Julia doc?

"To use the BridgeStan source you've manually downloaded instead of
one the package will download for you, you must use
[`set_bridgestan_path()`](#function-set-bridgestan-path) or the `$BRIDGESTAN`
environment "

Reply:
I would have pretty confidently claimed such a sentence already appeared on that page... I guess my Python favoritism is catching up with me. 

Will do.